lc-hax/Scripts/Helpers/Players.cs

```diff
@@ -62,4 +62,8 @@ internal static partial class Helper {
 
         return true;
     }
+    internal static bool IsDead(this PlayerControllerB? instance) => instance is not null && (!instance.isPlayerControlled || instance.health <= 0);
```

Comment:
```suggestion
    internal static bool IsDead(this PlayerControllerB player) => !player.isPlayerControlled;
```

Extensions usually never handle null instances and besides, we can just do:

```cs
player?.IsDead()
```

Reply:
True, i added it